java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java

```diff
@@ -740,6 +740,39 @@ private static String getBusId(Terminal t) {
                             .addStringSeries("bus_id", svc -> getBusId(svc.getTerminal())))
                             .build();
 
+                case VOLTAGE_LEVEL:
+                    List<VoltageLevel> voltageLevels = network.getVoltageLevelStream().collect(Collectors.toList());
+                    return addProperties(new SeriesPointerArrayBuilder<>(voltageLevels)
+                            .addStringSeries("id", VoltageLevel::getId)
+                            .addDoubleSeries("nominal_v", VoltageLevel::getNominalV)
+                            .addDoubleSeries("high_voltage_limit", VoltageLevel::getHighVoltageLimit)
+                            .addDoubleSeries("low_voltage_limit", VoltageLevel::getLowVoltageLimit))
+                            .build();
+
+                case SUBSTATION:
+                    List<Substation> substations = network.getSubstationStream().collect(Collectors.toList());
+                    return addProperties(new SeriesPointerArrayBuilder<>(substations))
+                            .addStringSeries("id", Identifiable::getId)
+                            .addStringSeries("tos", Substation::getTso)
```

Comment:
typo: it's TSO :) 
I propose to write it in capital letters (it's the acronym for Transmission System Operator)